src/eXpansion/Bundle/Dedimania/Resources/config/plugins.yml

```diff
@@ -0,0 +1,68 @@
+parameters:
+     expansion.dedimania.titles:
+            stadium:
+                - "TMStadium@nadeo"
+                - 'BaF1@mcrobert'
+                - 'Dirt_World_TM2@bernatf'
+                - 'Dirt_@mr.dvd'
+                - 'edenia@nexxusdrako'
+                - 'ESLTitlePack@nilakite2'
+                - 'Nations_Forever@citiroller'
+                - 'Nations_ESWC@tm-jinzo'
+                - 'Minimalize@mvv0105'
+                - 'only_stadium_car@adamkooo'
+                - 'RPG@tmrpg'
+                - 'SRE@tm-nascar'
+                - 'StadiumPlatform@darkpuddle_'
+                - 'Ultimate_Challenge_2@mr.dvd'
+                - 'esl_comp@lt_forever'
+            canyon:
+                -  "TMCanyon"
+                -  "TMCanyon@nadeo"
+                -  "Acrobatic@mr.dvd"
+                -  "_f00Canyon_00fStar@mcmart1"
+                -  'CanyonCity@darkpuddle_'
+                -  'Canyon_Inverted_edk@edk'
+                -  'DD_RailwaySystem@divingduck'
+                -  'Glide@darmaya'
+                -  'LEGO_Racing@macio6'
+                -  'POLSO@darmaya'
+                -  'Raid@meuh21'
+                -  'TM2_Canyon_Sparkstedition@sparkster'
+                -  'Mr.DvDCanyon_dvd@mr.dvd'
+                -  'Wastelands@mpmandark'
+                -  'wtc@woutre'
+            valley:
+                -  "TMValley"
+                -  "TMValley@nadeo"
+                -  'adrenalin@flighthigh'
+                -  'endless_Valley@flighthigh'
+                -  'F1_Abu_Dhabi@darkpuddle_'
+                -  'miniahoy3@kaeptniglu'
+                -  'RaidValley@meuh21'
+                -  'ValleyCity@darkpuddle_'
+                -  'Valley_Extensions@dag_bert'
+            lagoon:
+                - "TMLagoon"
+                - "TMLagoon@nadeo"
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: true
+
+    eXpansion\Bundle\Dedimania\Plugins\Dedimania:
+      class: eXpansion\Bundle\Dedimania\Plugins\Dedimania
+      arguments:
+          $titles: "%expansion.dedimania.titles%"
+          $enabled: "@expansion.dedimania.config.enable"
```

Comment:
what can be done is to have a data provider using the enable config. Then the plugin requires that data provider.

that way when it's disabled it's really disabled. And all plugins know about it. now they don't.